src/entity/learning-object/learning-object.ts

```diff
@@ -499,6 +509,16 @@ export class LearningObject {
     this.updateDate();
     return this._outcomes.push(addingOutcome) - 1;
   }
+  /**
+   * Adds a passed guideline to this learning object.
+   *
+   * @param guideline
+   * @returns
+   */
+  addGuideline(guideline: any) {
+    return this._guidelines.push(guideline) -1;
```

Comment:
Just curious, what does the -1 do?

Reply:
I'll be honest. I don't know. When Michael and I went through to add the guidelines array to the Learning Object type, we just followed how outcomes was implemented, and since it worked we didn't ask too many questions.